### PR TITLE
pip, pypi and requirements

### PR DESCRIPTION
**Pull request by robkooper**
- auto push to pypi
- cleanup of requirements
-

### CODE REVIEW

**Review by nunezco2 — APPROVED** [review]

**Comment by robkooper** [comment]
chatting with @jonglee1 , this will generate to restrictive requirements that will break installation of other packages.

File: .github/workflows/requirements.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: pyclowder
```
Comment (navarroc) [inline .github/workflows/requirements.yaml:1]: pyincore?

Comment (robkooper) [reply .github/workflows/requirements.yaml]: same copy and paste errorr.

**Review by navarroc — APPROVED** [review]

File: recipes/meta.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ package:
   version: {{ version }}
 
 source:
-  path: /path/to/pyincore
+  path: /Users/kooper/git/incore/pyincore
```
Comment (ywkim312) [inline recipes/meta.yaml:9]: can we change this to like ../ ?

**Review by nunezco2 — APPROVED** [review]

**Pull request merged**